### ISSUE TITLE
[WLED] send command "DECREASE or INCREASE" to Global_Brightness not working?

### ISSUE THREAD

**Issue by niceguy0815** [openhab/openhab-addons#14832]
Hi,
i’m trying to creat a rule for a Switch to dimm my WLED Binding /Thing

on my shelly and zigbee Lights / Dimmers i can use send command “DECREASE or INCREASE” to the Brightness item to get dimmer switch working but on the
" Global Brightness" item from the WLED Thing it is not working.

I see in the log:

` Item 'Kuchenzeile_WLED_Global_Brightness' received command INCREASE`

But it did not change the Brightness.

Than i hav testet it with the " Master Controls" Item or the " Segment Brightness" Item.

And on thes Items DECREASE command sems to worke but the INCREASE command shuts the WLED OFF and there is no steps to down dimming the Brightness.

IS this a bug ore do i something wrong?

example Rule DECREASE:

```
configuration: {}
triggers:
  - id: "1"
    configuration:
      event: "5002"
      channelUID: deconz:switch:00212E05388E:54ef4410004c1384010012:buttonevent
    type: core.ChannelEventTrigger
conditions: []
actions:
  - inputs: {}
    id: "2"
    configuration:
      itemName: Kuchenzeile_WLED_Global_Brightness
      command: DECREASE
    type: core.ItemCommandAction
```
or INCREASE:

```
configuration: {}
triggers:
  - id: "1"
    configuration:
      event: "3002"
      channelUID: deconz:switch:00212E05388E:54ef4410004c1384010012:buttonevent
    type: core.ChannelEventTrigger
conditions: []
actions:
  - inputs: {}
    id: "2"
    configuration:
      itemName: Kuchenzeile_WLED_Global_Brightness
      command: INCREASE
    type: core.ItemCommandAction
```

**Comment by markus7017** [comment]
Create a TRCE log, you should see a message like "Changing brightness from". 
Try to included it in the Main UI and to use the stepper and see if that makes a difference
Post the log section here: https://community.openhab.org/t/shelly-binding/

**Comment by markus7017** [comment]
no related to Shelly, sorry for confusion

**Comment by lsiepel** [comment]
What version of  openHAB is this issue related to? Ther ehave been some PR's that might be related, but they are a bit older 3.x

**Comment by Skinah** [comment]
The current merged binding in 4.1 Stable and older that uses a bridge/child things format uses JSON api for 99% of the features and falls back to the http API for INCREASE and DECREASE commands. I suspect what your seeing is a bug in the WLED firmware that is in the http API.

I just came across a poorly documented way to do this with the JSON methods:

Examples of frequently requested custom API:
```
<html><body>
<!--StartFragment-->
Increase brightness by 40 wrapping when maximum reached | {"bri":"w~40"}
-- | --


<!--EndFragment-->
</body>
</html>
```

I do not know if it is implemented or just requested, need someone to test and implement it if it works.

Also someone should test and report this on the main wled project github if it is a http api bug.
Lastly it would be helpful if people posted versions they are using. For example it may only be in older firmwares and the newer ones may have a fix.

**Comment by Skinah** [comment]
I can confirm that the {"bri":"w~40"} is not implemented in their JSON API as of the time of writting this, so if the http API has a bug, or it has been removed from the wled project, you need to report it to their github project. The reason I changed over to the json api was that none of the bugs in the http api that I reported were getting fixed.

I am going to close this issue unless someone contacts the WLED project and does the testing for a way that actually works, then feel free to let me know. Their documentation does state that this is commonly requested, so ask a few more times.
